### PR TITLE
Changes to support update import id.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/JdbcExamRepository.java [hunk 1]
```diff
@@ -53,6 +53,7 @@ public long create(final Exam exam) {
                 .addValue("exam_student_id", studentExamId)
                 .addValue("school_year", exam.getSchoolYear())
                 .addValue("import_id", exam.getImportId())
+                .addValue("upd_import_id", exam.getImportId())
```
Comment (chdietz) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/JdbcExamRepository.java]: upd vs update, where do we draw the line on abbreviation?
I prefer less abbreviation.

Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/JdbcExamRepository.java]: I really do not have a strong opinion here.

Comment (jtreuting) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/JdbcExamRepository.java]: If anyone asked me (which they didn't), i would go for "update_import_id".  Personally i don't mind longer variables or column names as often they are easier to understand

Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/JdbcExamRepository.java]: Ok, majority wins :) Will rename to ```update_import_id```.

Comment (bgsavage) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/JdbcExamRepository.java]: Should we rename import_id to created_import_id ?  Also I was thinking both should be past tense.

Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/JdbcExamRepository.java]: We intentionally chose not to change `import_id`. 

As for the tense, the way i read this is "the import_id associated with the update", not "the updated import_id".

**Comment by malaffoon** [comment]
I'm confused how the RDW_Schema pointer could be moved since the RDW_Schema changes aren't merged to develop yet.

**Comment by agorina** [comment]
@malaffoon , I was confused too. Maybe @rcordeau could clarify? I was not going to merge until I merge the schema, but I was wondering the same.

**Pull request merged**